content/_help/verify-your-identity/verify-your-address-by-mail._es.md

```diff
@@ -9,17 +9,17 @@ redirect_from:
 ---
 La verificación de la identidad es el proceso en el que acreditas que eres tú realmente y no alguien que se hace pasar por ti. Uno de los pasos de este proceso requiere que proporcione su número de teléfono principal para verificar su identidad.
 
-Si no puede verificarla por teléfono, le enviaremos una carta con un código único. Este código te permitirá terminar de verificar tu identidad y configurar la cuenta.
+Si no puede verificarla por teléfono, le enviaremos una carta con un código de verificación. Este código te permitirá terminar de verificar tu identidad y configurar la cuenta.
 
-La carta se envía a través del Servicio Postal de EE. UU. entre tres y siete días hábiles a partir del momento en que la solicitas. El código único caducará 30 días después de la fecha en que envíe su solicitud.
+La carta se envía a través del Servicio Postal de EE. UU. entre tres y siete días hábiles a partir del momento en que la solicitas. El código de verificación caducará 30 días después de la fecha en que envíe su solicitud.
```

Comment:
In the Spanish (but not French) app translations, we left the post office in English. "Las cartas se envían al siguiente día hábil por USPS First-Class Mail y tardan entre 5 y 10 días en llegar."

Reply:
Agree with this change to match changes made earlier.